### DISCUSSION TITLE
[2.1.3]sys mem available less than low water mark

### DISCUSSION THREAD

**Discussion by Chenfx-git** [apache/doris#38988] [A - General / Q&A]
![doris-1](https://github.com/user-attachments/assets/2a1bfd3e-ebbf-4e4e-930b-6570193022df)
![doris-2](https://github.com/user-attachments/assets/73a172e2-49ba-4699-9129-4b72a072fbf4)
![image](https://github.com/user-attachments/assets/dcee2fcc-f69b-490c-bb41-90c5f3bf6aa7)
Run in doris 2.1.3 test.
Only import data by logstash，no search.
Doris run for a time，this error occur. The service is unavailable.
Why sys mem so hight?

**Comment by xinyiZzz** [comment]
Hi @Chenfx-git 

upgrade Doris 2.1.5
or
Add `enable_use_cgroup_memory_info=false` in `be/conf/be.conf` and restart be
or
`http://xxxx:8040/api/update_config?enable_use_cgroup_memory_info=false` to modify dynamically

there is a bug in cgroup memory acquisition, fixed in 2.1.5, https://github.com/apache/doris/pull/36966

**Comment by Chenfx-git** [comment]
Add enable_use_cgroup_memory_info=false, still oom.
We use 2.1.5 version, it run well!
